client-core/src/wallet/syncer.rs

```diff
@@ -302,7 +302,21 @@ impl<'a, S: SecureStorage, C: Client, D: TxDecryptor> WalletSyncerImpl<'a, S, C,
     }
 
     fn sync(&mut self) -> Result<()> {
-        let status = self.env.client.status()?;
+        let mut counter = 0;
```

Comment:
I don't think we need automatic retries here.

- `PollingSynchronizer` should retry in case of failure.
- When manually calling sync (from `client-cli` or `client-rpc`), user can call it again after sometime.

Reply:
Ok, done.